<launch>
	<node pkg="odom_to_pose_converter" exec="odom_to_pose_converter_node" name="odom_to_pose_converter_node" output="screen">
		<remap from="odom_in" to="icp_odom" />
	</node>

	<node pkg="wiln" exec="wiln_node" name="wiln_node" output="screen">
		<remap from="pose_in" to="pose" />
		<param name="trajectory_speed" value="4.0" />
		<param name="low_pass_window_size" value="5" />
		<remap from="planned_trajectory" to="planned_trajectory_test" />
		<remap from="real_trajectory" to="real_trajectory_test" />
	</node>

</launch>
